libs/eo/emissions/shell/src/lib/eo-amissions-page-greenhouse-gasses.component.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-emissions-page-greenhouse-gasses',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+
+      mat-card {
+        img {
+          display: block;
+          width: 544px; // Magic number by designer
+          height: 133px; // Magic number by designer
+        }
+
+        h3 {
+          color: var(--watt-color-neutral-black);
+        }
```

Comment:
I would like to remove this styling, since the H3 element will be updated in WATT at some point :)

Reply:
Will that be black by default?

Reply:
Not right now no, it will be a greenish tinted color :)

Reply:
But when done, will it be black then? If so, then these are not needed, I agree :-)